app/streams/publishing_api/message_processor.rb

```diff
@@ -22,24 +19,21 @@ def process
         # RabbitMQ guarantees at-least-once delivery when using acknowledgements,
         # so messages can be duplicated, plus we can't guarantee the order of
         # processing because we can have multiple consumers working in parallel.
-        logger.info "Skipping message from publishing API: published with #{payload_version} but we've already received a message with #{item.publishing_api_payload_version}"
+        logger.info "Skipping message from publishing API: published with #{incoming_content_item.payload_version} but we've already received a message with #{item.publishing_api_payload_version}"
       end
     else
       Dimensions::Item.create_empty(
```

Comment:
This case seems wrong as it doesn't run the importer

Reply:
Have this tracked in https://trello.com/c/Yqb5YxIq/449-story-template-take-copy now.